src/opm/io/eclipse/ExtSmryOutput.cpp

```diff
@@ -148,6 +148,24 @@ std::vector<std::string> ExtSmryOutput::make_modified_keys(const std::vector<std
 
             mod_keys.push_back(mod_key);
 
+        } else if (valueKeys[n].substr(0,1) == "R"){
+            std::string str34 = valueKeys[n].substr(2,2);
+            std::string str45 = valueKeys[n].substr(3,2);
+
+            if (valueKeys[n].substr(0,5) == "RORFR"){
+                mod_keys.push_back(valueKeys[n]);
+
+            } else if ((str34 == "FR") || (str34 == "FT") || (str45 == "FR") || (str45 == "FT")) {
+                auto p = valueKeys[n].find(":");
+                int num = std::stoi(valueKeys[n].substr(p+1));
```

Comment:
Do we need to account for the possibility that there might not be a colon (`:`) in the value key?  If so, we should add a check that
```C++
p != std::string::npos
```
before using `p + 1` in `substr()` to extract the `num` value here.

I ask because if the only source of input here, i.e. the input that creates `valueKeyes`, is code that we control then we don't need the check.  If on the other hand the input could be from an untrusted source then it's probably better to be defensive.